alpha/lib/model/UserLoginDataPeer.php

```diff
@@ -1011,7 +1011,7 @@ public static function getCacheInvalidationKeys()
 		return array(array("userLoginData:id=%s", self::ID), array("userLoginData:loginEmail=%s", self::LOGIN_EMAIL));		
 	}
 
-	public static function getAuthInfoLink($hashKey)
+	public static function getAuthInfoLink($hashKey, $dynamicBaseQrLink = null)
```

Comment:
just call it qrLink and write the code in the following way, no need to set it explicitly 

if($qrLink)
{
			$resetLinksArray = kConf::get('password_reset_links');
			$qrLink = $resetLinksArray['qr_page'];
}